packages/graphqlsp/src/ast/resolve.ts

```diff
@@ -34,37 +36,33 @@ export function resolveTemplate(
   const resolvedSpans = node.template.templateSpans
     .map(span => {
       if (ts.isIdentifier(span.expression)) {
-        const definitions = info.languageService.getDefinitionAtPosition(
-          filename,
-          span.expression.getStart()
-        );
-
-        const def = definitions && definitions[0];
-        if (!def) return;
+        const typeChecker = info.languageService.getProgram()?.getTypeChecker();
```

Comment:
This is awkwardly repeated now across the codebase. We can manually refactor our helper functions to just accept the `typeChecker` directly now afaict